jina/peapods/pods/__init__.py

```diff
@@ -750,7 +745,6 @@ def _set_peas_args(args: Namespace) -> Dict[int, List[Namespace]]:
             for replica_id in range(args.replicas):
                 _args = copy.deepcopy(args)
                 _args.shard_id = shard_id
-                _args.replica_id = replica_id
                 _args.pea_role = PeaRoleType.WORKER
                 _args.identity = random_identity()
 
```

Comment:
It doesn't let me comment directly there, but just below here is some more code that seems to be related to replica_id, but I am not sure what it does. Does it need to be removed/changed?

Reply:
does not seem so